### ISSUE TITLE
Module parse failed: 'import' and 'export' may appear only with 'sourceType: module'

### ISSUE THREAD

**Issue by trevorpfiz** [sanity-io/ui#1037]
Updated to `1.0.6` from `1.0.5`.

```
./node_modules/@sanity/ui/dist/index.esm.js
Module parse failed: 'import' and 'export' may appear only with 'sourceType: module' (61:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| function _toPropertyKey(arg) { var key = _toPrimitive(arg, "string"); return typeof key === "symbol" ? key : String(key); }
| function _toPrimitive(input, hint) { if (typeof input !== "object" || input === null) return input; var prim = input[Symbol.toPrimitive]; if (prim !== undefined) { var res = prim.call(input, hint || "default"); if (typeof res !== "object") return res; throw new TypeError("@@toPrimitive must return a primitive value."); } return (hint === "string" ? String : Number)(input); }
> import { jsx, jsxs, Fragment } from 'react/jsx-runtime';
| import { useMemo, useState, useRef, useEffect, createContext, useContext, useLayoutEffect, forwardRef, useId, useCallback, cloneElement, isValidElement, createElement, Component, memo, useReducer, Children, Fragment as Fragment$1 } from 'react';
| import ReactIs, { isElement as isElement$1, isFragment, isValidElementType } from 'react-is';

Import trace for requested module:
./node_modules/@sanity/ui/dist/index.esm.js
./node_modules/sanity/lib/index.esm.js
./sanity.config.ts
./app/(studio)/studio/[[...index]]/page.tsx
```

**Comment by estubmo** [comment]
Having the exact same issue

**Comment by geball** [comment]
Sorry you're encountering this error. As @ElektrikSpark mentioned, this is indeed a regression in `@sanity/ui@1.0.6` and the Engineering team is now aware of it. In the meantime, you should be able to work around the error by adding the following to your package.json, deleting your node_modules and lock file, and then re-installing:

## If using npm

"overrides": {
  "@sanity/ui": "1.0.5"
}

## If using yarn

```json
"resolutions": {
  "@sanity/ui": "1.0.5"
}
```

**Comment by vincentjflorio** [comment]
Happy to confirm a quick resolution after implementing this fix.

**Comment by modecode22** [comment]
thanks

**Comment by benja** [comment]
To build on what @geball said (thanks!), for **pnpm** users:

```json
"pnpm": {
  "overrides": {
    "@sanity/ui": "1.0.5"
  }
}
```

**Comment by mariuslundgard** [comment]
Sorry for the inconvencience. This issue has been fixed as of `@sanity/ui@1.0.7`.
